datasets/at/nbter_sanctions/at_nbter_sanctions.yml

```diff
@@ -33,7 +36,7 @@ assertions:
   min:
     schema_entities:
       Person: 40
-      Organization: 25
+      Organization: 20
```

Comment:
is this how many entities we'd expect when all the dupes are merged? It's probably sensible to guestimate how many unique entities there are when they're all deduped, and use that for the minimum requirements

Reply:
we have roughly 17 unique entities, I'll set it to this amount then